### PR TITLE
Multiple updates to cordova plugin search

### PR DESCRIPTION
**Pull request by nikhilkh**
* Faster gulp live reload workflow for plugin search work
* Add sortBy URL parameter on sorting
* Tweak title and description meta tags

### CODE REVIEW

**Comment by nikhilkh** [comment]
@riknoll Please review

**Comment by riknoll** [comment]
Checking multiple platform filters results in an improperly encoded URL. I have a fix, do you want me to push it?

File: gulpfile.js [hunk 1]
```diff
@@ -121,7 +121,7 @@ gulp.task("watch", ["serve"], function () {
             path.join(ROOT_DIR, "**", "*.yml"),
             path.join(SOURCE_DIR, "**", "*.html"),
             path.join(SOURCE_DIR, "**", "*.md"),
-            path.join(JS_DIR, "**", "*.js"),
+            path.join(JS_DIR, "**", "*.js" + "!" + PLUGINS_FILE_NAME),
```
Comment (dblotsky) [inline gulpfile.js]: What's the motivation behind excluding this file?

File: gulpfile.js [hunk 2]
```diff
@@ -234,8 +234,7 @@ gulp.task("plugins", function() {
         //           minified JS has some things that look like
         //           Liquid tags, so we replace them manually
         .pipe(replace("){{", "){ {"))
-
-        .pipe(gulp.dest(JS_DIR));
+        .pipe(gulp.dest(JS_DIR))
```
Comment (dblotsky) [inline gulpfile.js]: Consistency nit pick: missing semicolon.

File: gulpfile.js [hunk 3]
```diff
@@ -224,7 +223,8 @@ gulp.task("plugins", function() {
     }
 
     return stream
-
+        .pipe(browsersync.reload({stream: true}))
+        .pipe(gulp.dest(JS_DIR.replace(SOURCE_DIR, out_dir)))
```
Comment (dblotsky) [inline gulpfile.js]: I think these should be in the reverse order: first replacing the file and then reloading the browser.

**Comment by riknoll** [comment]
Updating a file in the plugins folder causes all image links in the plugins page to be broken once the page refreshes (and on all subsequent refreshes)

**Comment by riknoll** [comment]
The issue is that `{{ site.baseurl }}` strings need to be processed by Jekyll and the updated gulpfile bypasses that

**Comment by dblotsky** [comment]
Also, it might be useful to switch to a proper URI builder for constructing URIs now.

**Comment by riknoll** [comment]
@dblotsky I addressed your concerns with Gulpfile. Gulp will now reload the browser twice on plugins changes. The first will update the code and the second will be a full push that includes updated `{{ site.baseurl }}` links.

**Comment by riknoll** [comment]
And yes, we should switch to a URI builder at some point.

**Comment by dblotsky** [comment]
Merged. Feel free to close.

**Comment by nikhilkh** [comment]
Thanks Richard for making the fixes!

**Pull request closed without merging**